internal/pkg/argocd/argocd.go

```diff
@@ -74,8 +78,9 @@ func generateArgocdAppDiff(ctx context.Context, app *argoappv1.Application, proj
 		}
 
 		ignoreAggregatedRoles := false
+		ignoreNormalizerOpts := normalizers.IgnoreNormalizerOpts{}
 		diffConfig, err := argodiff.NewDiffConfigBuilder().
-			WithDiffSettings(app.Spec.IgnoreDifferences, overrides, ignoreAggregatedRoles).
+			WithDiffSettings(app.Spec.IgnoreDifferences, overrides, ignoreAggregatedRoles, ignoreNormalizerOpts).
```

Comment:
ignoreNormalizerOpts - this is needed for the newer version of ArgoCD client, this is not related to change we are making here